drake/CMakeLists.txt

```diff
@@ -162,11 +162,14 @@ file(WRITE ${CMAKE_INSTALL_PREFIX}/matlab/addpath_drake.m
 	   "  wd = cd(fullfile(mfiledir,'${relpath}'));\n"
 	   "  addpath_drake();\n"
 	   "  cd(wd);\n"
+     "end\n"
+     "\n"
     )
 
 file(WRITE ${CMAKE_INSTALL_PREFIX}/matlab/rmpath_drake.m
 	   "function rmpath_drake()\n"
 	   "  mfiledir = fileparts(which(mfilename));\n"
+     "  jarfiledir = fullfile(mfiledir, '..', 'share', 'java');\n"
```

Comment:
whoops, that line shouldn't be there. Thanks!

And rmpath_drake (the one living in drake/) already removes the drake.jar from the classpath.